server/scsynth/scsynth_main.cpp

```diff
@@ -126,6 +158,10 @@ void Usage() {
 
 
 int scsynth_main(int argc, char** argv) {
+#ifdef _WIN32
+    std::thread displayWarningIfScsynthHasNotBootedThread(displayWarningIfScsynthHasNotBooted);
```

Comment:
this is kind of a mouthful! it makes sense to keep long names for functions at file-level scope, but for locally scoped variables you can be more concise. could i recommend `bootWarningThread`?